### PR TITLE
Rick cogley patch 1

### PR DESCRIPTION
**Pull request by RickCogley**
Removed extra spaces, which I had added after reading the RST spec, because they don't work anyway. My motivation for adding them was, some of the lines are mashed together and I was trying to force a carriage return in certain places.

### CODE REVIEW

File: docs/turn-howto.rst [hunk 1]
```diff
@@ -9,15 +9,17 @@ the Home Server to generate credentials that are valid for use on the TURN
...
 and integrate it with synapse.
 
 coturn Setup
 ============
 
+You may be able to setup coturn via your package manager,  or set it up manually using the usual ``configure, make, make install`` process.  
+
  1. Check out coturn::
-      svn checkout http://coturn.googlecode.com/svn/trunk/ coturn
+      svn checkout https://github.com/coturn/coturn.git coturn
```
Comment (dbkr) [inline docs/turn-howto.rst]: I appreciate github supports svn, but if we're switching to a git repository I think it makes sense to check it out with git: this is likely to confuse people.

**Comment by dbkr** [comment]
Other than that, looks good, thanks for updating this.

**Comment by RickCogley** [comment]
I agree. It should be ``git clone``. By the above are you saying you'll commit it making that change or, do you want me to do it?

**Comment by dbkr** [comment]
If you can edit this on your branch, that's slightly easier since otherwise I'd have to merge this then change it afterwards.

Also I've just noticed this is against master rather than develop, so you'll need to make a new PR against the right branch I'm afraid. You'll need to sign off on the DCO too: the details are all in https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.rst

**Comment by RickCogley** [comment]
Of course. Ugh, sorry about the trouble. I'll do that.

**Comment by RickCogley** [comment]
Closing; Cleaned up in: https://github.com/matrix-org/synapse/pull/900

**Pull request closed without merging**